concourse/scripts/ssh.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eu
```

Comment:
We could do with `-o pipefail` here as well. It's also probably more reliable to have a dedicated `set` line instead of relying on the `#!` line. That way they'll apply however this script is invoked (an be consistent with other scripts).